pkg/build/tls.go

```diff
@@ -25,12 +25,32 @@ import (
 )
 
 const (
-	certificateOrgName = "cnoe.io"
+	certificateOrgName     = "cnoe.io"
+	certificateValidLength = time.Hour * 8766
```

Comment:
Not really a big deal, but maybe we could shorten this? I feel like folks will not be keeping builder clusters up for more than weeks, but just in case we should flag the need to rotate these carts sooner?

Reply:
Yeah we can shorten it. Will follow up with another PR.